international-labour-organization.csl

```diff
@@ -109,15 +113,15 @@
     </choose>
   </macro>
   <macro name="pages">
-    <group>
-      <label variable="page" form="short" suffix=" "/>
+    <group prefix="p">
+      <label suffix=" " variable="page" form="short"/>
       <text variable="page"/>
     </group>
```

Comment:
@vl2, this is also not correct. `<label suffix=" " variable="page" form="short"/>` should already give "p. ". If you want "pp.", the proper way is to use the plural form of the label term, i.e. `<label variable="page" form="short" plural="true" suffix=" "/>`, and remove the `prefix="p"` from the group.